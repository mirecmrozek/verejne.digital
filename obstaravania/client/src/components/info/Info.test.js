import React from 'react';
import renderer from 'react-test-renderer';
import Info from './Info';

it('renders dummy as before', () => {
  const dummy = {"zrsr_data":[],"company_stats":[],"contracts":[],"new_orsr_data":[],"sponzori_stran_data":[],"related":[{"name":"Dummy, a.s.","eid":123,"address":"Dummy street 123, Bratislava","lat":123,"lng":123,"ds":[null,0.123,1,0],"size":1}],"auditori_data":[],"audiovizfond_data":[],"entities":[{"lat":"123","lng":"123","entity_name":"Dummy","address":"Dummy Street 123, 851 04 Petr≈æalka, Slovakia"}],"firmy_data":[],"total_contracts":0,"advokati_data":[],"nadacie_data":[],"orsresd_data":[],"stranicke_prispevky_data":[{"strana":"Dummy","rok":"2010","mena":"EUR","vyska_prispevku":"123"}],"uzivatelia_vyhody_ludia_data":[]}; // eslint-disable-line
  const component = renderer.create(<Info data={dummy} />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
