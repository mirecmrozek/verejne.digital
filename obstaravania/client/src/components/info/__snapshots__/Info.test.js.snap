exports[`test renders dummy as before 1`] = `
<div
  className="result">
  <span
    className="entityName">
    <i
      aria-hidden="true"
      className="fa fa-circle" />
     
    Dummy
     
    <a
      href="?zobraz&123&123&undefined&"
      rel="noopener noreferrer"
      target="_blank"
      title="Zobraz na mape">
      <i
        aria-hidden="true"
        className="fa fa-map-marker" />
    </a>
  </span>
  <hr />
  <table
    className="dataTable table table-condensed">
    <tbody>
      <tr>
        <td
          colSpan="2">
          Dummy Street 123, 851 04 Petržalka, Slovakia
        </td>
      </tr>
      <tr>
        <td
          colSpan="2">
          <strong>
            Stranícke príspevky:
          </strong>
          <ul>
            <li>
              <a
                className={false}
                href="http://datanest.fair-play.sk/searches/quick?query_string=Dummy"
                rel="noopener noreferrer"
                target="_blank">
                Dummy, 123 EUR (rok 2010)
                 
                <i
                  aria-hidden="true"
                  className="fa fa-external-link" />
              </a>
            </li>
          </ul>
        </td>
      </tr>
      <tr>
        <td>
          <button
            className="showHideBtn btn btn-link"
            onClick={[Function]}>
            Vzťahy
          </button>
        </td>
        <td
          className="sizeCell">
          <strong>
            1
          </strong>
          <i
            aria-hidden="true"
            className="fa fa-chevron-down sizeCellArrow"
            onClick={[Function]} />
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
